src/framework/genesis_framework.gemspec

```diff
@@ -8,11 +8,12 @@ Gem::Specification.new do |gem|
   gem.email = 'opensourcesoftware@tumblr.com'
 
   gem.date = '2016-01-13'
-  gem.version = '0.6.8'
+  gem.version = '0.6.10'
```

Comment:
Because I really hope https://github.com/tumblr/genesis/pull/69 will be merged 1st :p